engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/selectModulesScreen/SelectModulesScreen.java

```diff
@@ -570,6 +571,7 @@ private void updateModuleInformation() {
                 info.setOnlineVersion(remote);
             }
         }
+        filterModules();
```

Comment:
This one fixes the problem with the list updating after the remote modules have downloaded.